ex03.ex

```diff
@@ -101,8 +106,9 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal([a | t1], [a | t2]), do: list_equal(t1, t2)
```

Comment:
I'm not deducting here, but `list_equal("cat", [1, 2]])` would return `false`, whereas is should probably raise an error.